bin/triton-mysql.py

```diff
@@ -175,7 +175,7 @@ def __init__(self):
         self.user = os.environ.get('MANTA_SUBUSER', None)
         self.role = os.environ.get('MANTA_ROLE', None)
         self.key_id = os.environ.get('MANTA_KEY_ID', None)
-        self.private_key = os.environ.get('MANTA_PRIVATE_KEY')
+        self.private_key = os.environ.get('MANTA_PRIVATE_KEY').replace('#', '\n')
```

Comment:
This was caused by `MANTA_PRIVATE_KEY` being unset in the `docker-compose.yml` file. That was fixed in https://github.com/autopilotpattern/mysql/pull/14/commits/9239d16a5c023764f1c00816eee7954a9cd3ef9a